sql/preprocessing.sql

```diff
@@ -23,19 +23,14 @@ RENAME COLUMN wkb_geometry TO geom;
 ALTER TABLE dcp_zoningmapindex 
 RENAME COLUMN wkb_geometry TO geom;
 
-DROP TABLE IF EXISTS dcp_zoning_taxlot;
-
-ALTER TABLE dcp_zoningtaxlots
-RENAME TO dcp_zoning_taxlot;
-
 DROP TABLE IF EXISTS dof_dtm_tmp;
 CREATE TABLE dof_dtm_tmp as(
     SELECT 
         bbl, 
         COALESCE(boro::text,LEFT(bbl::text, 1)) as boro, 
         COALESCE(block::text, SUBSTRING(bbl::text, 2, 5)) as block,
         COALESCE(lot::text, SUBSTRING(bbl::text, 7, 4)) as lot,
-        ST_Multi(ST_Union(f.geom)) as geom
+        ST_Multi(ST_Union(ST_MakeValid(f.geom))) as geom
```

Comment:
This is the most interesting change in the PR (to me) - I would need to do some more digging but I wonder why we wouldn't want to always make a geometry valid via the `ST_MakeValid` especially if we are dealing with mixed geom types

Reply:
Looking at this documentation for ref: https://postgis.net/docs/ST_MakeValid.html

Reply:
I wonder if this would solve our issues with the data-library ingestion as well

Reply:
for reference, I added it to fix this [error in a build](https://github.com/NYCPlanning/db-zoningtaxlots/actions/runs/4442534719/jobs/7798855332#step:6:28): 
```
Could not create GEOS COLLECTION from geometry array: IllegalArgumentException: Invalid number of points in LinearRing found 3 - must be 0 or >= 4
```

we do seem to be using `makeValid=True` during [data-library ingestion](https://github.com/NYCPlanning/db-data-library/blob/2cf6f29d2f1b1c52383e6a9aaabbe08f8dd1486f/library/ingest.py#L111), so I may only have had to use it because the `ogr2ogr` -> `shp2pgsql` combo replaced `gdal.VectorTranslate`

and maybe that error above is more evidence that the source data has some unexpected geometries in it??